configure.ac

```diff
@@ -267,29 +254,104 @@ dnl --- enable_pkey
 enable_pkey=no
 
 case $target in
-     *s390x*)
-	  enable_pkey=yes
-	  ;;
-     *s390*)
-	  CFLAGS="$CFLAGS -m31 -DNO_PKEY"
-	  ;;
-     *ppc64* | *x86_64*)
-	  CFLAGS="$CFLAGS -DNO_PKEY"
-	  ;;
-     *ppc* | i*86*)
-	  CFLAGS="$CFLAGS -m32 -DNO_PKEY"
-	  ;;
-     *)
-	  CFLAGS="$CFLAGS -DNO_PKEY"
-	  ;;
+	*s390x*)
+		enable_pkey=yes
+	;;
+	*s390*)
+		CFLAGS="$CFLAGS -m31"
+		AC_DEFINE([NO_PKEY])
+	;;
+	*ppc64* | *x86_64*)
+		AC_DEFINE([NO_PKEY])
+	;;
+	*ppc* | i*86*)
+	  CFLAGS="$CFLAGS -m32"
```

Comment:
This line is indented differently than the others.

Reply:
Fixed - I think I kept forgetting to persist these changes in my patch file, overwriting it with other changes. Should be fixed for good now!